src/api/providers/ollama.ts

```diff
@@ -24,7 +25,18 @@ export class OllamaHandler implements ApiHandler {
 	private ensureClient(): Ollama {
 		if (!this.client) {
 			try {
-				this.client = new Ollama({ host: this.options.ollamaBaseUrl || "http://localhost:11434" })
+				const clientOptions: Partial<Config> = {
+					host: this.options.ollamaBaseUrl || "http://localhost:11434",
```

Comment:
```suggestion
					host: this.options.ollamaBaseUrl,
```
If the this.options.ollamaBaseUrl is not defined, we can just use undefined so that it would fallback to using the default endpoint in user's local setting instead of setting it to `http://localhost:11434` that might not work in all OS?

Reply:
this was previously just the default used when running ollama locally